generated/nise/nise/_library_singleton.py

```diff
@@ -12,7 +12,7 @@
 
 _instance = None
 _instance_lock = threading.Lock()
-_library_info = {'Linux': {'64bit': {'name': 'nise', 'type': 'cdll'}},
+_library_info = {'Linux': {'64bit': {'name': 'libnise.so', 'type': 'cdll'}},
```

Comment:
I think this is a mistake, at least the way things work it should say `nise`.

Reply:
Fixed